pages/projects/[id]/index.js

```diff
@@ -0,0 +1,106 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import Image from "next/image";
+import { FaArrowLeftLong } from "react-icons/fa6";
+import { uid } from "uid";
+import styled from "styled-components";
+
+export default function ProjectDetailsPage({ projects }) {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const projectData = projects.find((project) => project.id === id);
+
+  if (!projectData) {
+    return null;
+  }
+
+  const {
+    title,
+    imageUrl,
+    complexity,
+    description,
+    duration,
+    materials,
+    steps,
+  } = projectData;
+
+  return (
+    <>
+      <Link href="/">
+        <FaArrowLeftLong /> Back
+      </Link>
+
+      <StyledDetailsWrapper>
+        <h2>{title}</h2>
+        <StyledImageWrapper>
+          <Image src={imageUrl} alt={title} width={300} height={200} />
+          <StyledP>{complexity}</StyledP>
+        </StyledImageWrapper>
+        <p>{description}</p>
+        <StyledDuration>Duration: {duration}</StyledDuration>
+
+        <StyledMaterialsList>
+          <h3>Materials</h3>

```

Comment:
A list should only have list-items as direct children. You can place the headline above the list.